Services/UI/OJS.Services.Ui.Data/ISubmissionsDataService.cs

```diff
@@ -1,3 +1,5 @@
+using OJS.Services.Ui.Models.Submissions;
```

Comment:
move after the namespace

Reply:
Removed it altogether due to not being used.